src/coreclr/jit/compiler.h

```diff
@@ -8732,7 +8758,7 @@ XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX
     unsigned getSIMDVectorRegisterByteLength()
     {
 #if defined(TARGET_XARCH)
-        if (getSIMDSupportLevel() == SIMD_AVX2_Supported)
+        if (getSIMDSupportLevel() >= SIMD_AVX2_Supported)
```

Comment:
Yes. If we plan for `Vector<T>` to support `V512` then we'd need to update this place to be `ZMM_REGSIZE_BYTES`.

Reply:
Yes, eventually once we have `Vector<T>` updated to support 512. My assumption was that we want to have this return  `ZMM_REGSIZE_BYTES` till then since this method is used only for `Vector<T>` I'll add a `// TODO-XArch-AVX512` here

Reply:
Done